examples/2_Iterative/01_optimisation_gd_fista.ipynb

```diff
@@ -690,9 +691,7 @@
    "source": [
     "Run the algorithm for a specified number of iterations with increased verbosity/amount of printing to screen.\n",
     "\n",
-    "Here we run for 5000 iterations to see a very well converged primal-dual gap. In practice fewer iterations are often sufficient for a visually converged image.\n",
-    "\n",
-    "This will take approximatly 8 minutes to run."
+    "Here we initially run for 500 iterations."
```

Comment:
ALso nice to show that fewer iterations suffice, I usually say this, but better that the notebook shows it like this.